### ISSUE TITLE
BeSman command "bes create -env <> -V <> basic  " not accepting  "-"(hyphen)

### ISSUE THREAD

**Issue by Aditi8007** [Be-Secure/BeSman#132]
### Current Behavior : 
While creating new environment with below command
`bes create -env jackson-core-RT-env -V 0.0.1 basic`
script is not allowing "-"(hyphen) so it creating directly by omitting "-" (Hyphen)

output is given below
```
aditi8007@L-357009166:~/Envfiles$ bes create -env jackson-core-RT-env -V 0.0.1 basic
Variable 'BESMAN_LOCAL_ENV' value updated to 'true'
Creating env file..
Creating new config file /home/aditi8007/Envfiles/jackson/0.0.1/besman-jackson-core-env-config.yaml
grep: /home/aditi8007/Envfiles/list.txt: No such file or directory
Updating local list
```
### Impact:
 so the variable "BESMAN_ARTIFACT_URL" inside the config file "besman-jackson-core-env-config.yaml" is written wrongly.

`BESMAN_ARTIFACT_URL: https://github.com/Be-Secure/jackson`
 (This is wrong github URL,right github url is https://github.com/Be-Secure/jackson-core)

### Expected Output:
`BESMAN_ARTIFACT_URL: https://github.com/Be-Secure/jackson-core`

### Solution:
Please modify the Script accordingly so that `BESMAN_ARTIFACT_URL` is exactly how it is in GitHub

**Comment by Aditi8007** [comment]
FYI @samirparhi-dev

**Comment by asa1997** [comment]
@Aditi8007 There shouldnt be hyphens in the project name. It should be underscores or it breaks the code.

**Comment by asa1997** [comment]
@pramit-d  Error while installing env scripts with hyphen in project names

**Comment by asa1997** [comment]
Closed with #139
